### DISCUSSION TITLE
support for apple tv

### DISCUSSION THREAD

**Discussion by nieyanzhai** [flet-dev/flet#4775] [Ideas]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

When there will be support for apple tv, is there a plan?

**Comment by FeodorFitsner** [comment]
Apple TV is not yet officially supported by Flutter. There are some hacky ways to make Flutter work with tvOS: https://github.com/flutter/flutter/issues/47928 - I haven't tried doing that. If you can investigate (i.e. successfully run Flutter on tvOS) then it would be helpful and we could maybe use your work to support tvOS.
